src/services/user/adminUnblockUser.js

```diff
@@ -1,12 +1,14 @@
 const {successResponse, errorResponse} = require('../../utils');
 const {adminUnblockUserProcess} = require('../../processes/scoring/admin-block-unblock-user');
+const UsersFunction = require('../../databases/functions/users');
 
 const adminUnblockUser = async (req, res) => {
   try {
     const {userId} = req.body;
     if (!userId) {
       return errorResponse(res, 'userId is required', 400);
     }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Error handling for invalid or non-existent `userId` is good, but there should also be error handling for when the `userUnblockByAdmin` function fails. This could be due to database issues or other unforeseen circumstances. Consider wrapping the call to `userUnblockByAdmin` in a try-catch block and return an appropriate error response if it fails.
```diff
    await UsersFunction.userUnblockByAdmin(userId);
+    try {
+      await UsersFunction.userUnblockByAdmin(userId);
+    } catch (error) {
+      return errorResponse(res, 'Failed to unblock user by admin', 500);
+    }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->